app/kumactl/cmd/root.go

```diff
@@ -77,6 +78,7 @@ func NewRootCmd(root *kumactl_cmd.RootContext) *cobra.Command {
 	cmd.PersistentFlags().StringVarP(&root.Args.Mesh, "mesh", "m", "default", "mesh to use")
 	cmd.PersistentFlags().StringVar(&args.logLevel, "log-level", kuma_log.OffLevel.String(), kuma_cmd.UsageOptions("log level", kuma_log.OffLevel, kuma_log.InfoLevel, kuma_log.DebugLevel))
 	cmd.PersistentFlags().BoolVar(&args.noConfig, "no-config", false, "if set no config file and config directory will be created")
+	cmd.PersistentFlags().DurationVar(&root.Args.ApiTimeout, "api-timeout", time.Minute, "the timeout for api calls")
```

Comment:
Since usually in http client we configure connection & read timeout I'd also add `The timeout includes connection time, any redirects, and reading the response body.` and maybe `A Timeout of zero means no timeout`.

https://pkg.go.dev/net/http#Client